.github/workflows/run-tests-rvv.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2020 - 2022 Free Software Foundation, Inc.
+#
+# This file is part of VOLK
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+#
+
+name: Run VOLK tests on different RVV configurations
+
+on: [push, pull_request]
+
+jobs:
+  Tests:
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: "recursive"
+    - name: Install packages
+      run: |
+        sudo apt-get update -q -y
+        sudo apt-get install -y python3-mako cmake qemu-user-static g++-14-riscv64-linux-gnu clang-18
+        mkdir build
+        cd build
+    - name: Test gcc-14 VLEN=128
+      run: |
+        cd build; rm -rf *
+        CXX=riscv64-linux-gnu-g++-14 CC=riscv64-linux-gnu-gcc-14 VLEN=128 \
+        cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchains/rv64gcv-linux-gnu.cmake ..
+        make -j$(nproc)
+        ARGS=-j$(nproc) make test
```

Comment:
I just noticed that this tests run without the `verbose` flag. That'd be great. e.g. one would get a bit more information which kernels are actually executed in each test.
A second little thing: looking at all the other CI tests, we use `ctest -V`. Since these tests usually run rather quickly, it is nice to have them run sequentially. Since the whole CI test here is reported to run for ~5min, I assume this `-j$(nproc)` doesn't contribute much to the overall runtime.